cmd/fabric-ca-client/command/main_test.go

```diff
@@ -2531,7 +2531,11 @@ func registerAndRevokeUsers(t *testing.T, admin *lib.Identity, num int) []*big.I
 			// Reenroll the user, this should create a new certificate, so this
 			// user will have two valid certificates, but we will revoke one
 			// of her certificate only
-			_, err := enrollResp.Identity.Reenroll(&api.ReenrollmentRequest{})
+			_, err := enrollResp.Identity.Reenroll(&api.ReenrollmentRequest{
+				CSR: &api.CSRInfo{
+					KeyRequest: &api.KeyRequest{},
```

Comment:
This should be optional, right? So why did the test need to change?

It's likely I'm missing something.

Reply:
Fixed